release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -157,7 +157,7 @@ For more information, see xref:../installing/installing_gcp/installing-gcp-accou
 ==== `propagateUserTags` parameter for AWS resources provisioned by the {product-title} cluster
 In {product-title} {product-version}, the `propagateUserTags` parameter is a flag that directs in-cluster Operators to include the specified user tags in the tags of the AWS resources that the Operators create.
 
-For more information, see xref:../installing/installing_aws/installing-aws-network-customizations.html#installation-configuration-parameters_installing-aws-network-customizations[Optional configuration parameters].
+For more information, see xref:../installing/installing_aws/installing-aws-network-customizations.adoc#installation-configuration-parameters_installing-aws-network-customizations[Optional configuration parameters].
```

Comment:
@aireilly I suggest changing the .adoc back to .html since asciidoc is rendered to html.